keps/sig-release/3000-artifact-distribution/README.md

```diff
@@ -0,0 +1,192 @@
+
+The cost of distributing Kubernetes comes at great cost nearing $150kUSD/month (mostly egress) in donations.
+
+Additionally some of our community members are unable to access the official release artifacts due to country level firewalls that do not them connect to Google services.
+
+Ideally we can dramatically reduce cost and allow everyone in the world to download the artifacts released by our community.
+
+We are now used to using the [image promoter process](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/1734-k8s-image-promoter) to promote images to the official kubernetes container registry using the infrastructure (GCR staging repos etc) provided by [sig-k8s-infra](https://github.com/kubernetes/k8s.io/tree/main/k8s.gcr.io)
+
+## Why a new domain?
+
+So far we (all kubernetes project) are using GCP as our default infrastructure provider for all things like GCS, GCR, GKE based prow clusters etc. Google has graciously sponsored a lot of our infrastructure costs as well. However for about a year or so we are finding that our costs are sky-rocketing because the community usage of this infrastructure has been from other cloud providers like AWS, Azure etc. So in conjunction with CNCF staff we are trying to put together a plan to host copies of images and binaries nearer to where they are used rather than incur cross-cloud costs.
+
+One part of this plan is to setup a proxy OCI service, that can identify where the traffic is coming from and redirect to the nearest image layer/repository. This is why we are setting up a new service using what we call an [oci-proxy](https://github.com/kubernetes-sigs/oci-proxy) for everyone to use. This proxy will identify traffic coming from, for example, a certain AWS region, then will setup a HTTP redirect to a source in that AWS region. If we get traffic from GKE/GCP or we don't know where the traffic is coming from, it will still redirect to the current infrastructure (k8s.gcr.io).
+
+## How can we help?
+
+When Kubernetes master opens up for v1.25 development, we need to update all default urls in our code and test harness to the new registry url. As a team sig-k8s-infra is signing up to ensure that this oci-proxy based registry.k8s.io will be as robust and available as the current setup. As a backup, we will continue to run the current k8s.gcr.io as well. So do not worry about that going away. Turning on traffic to the new url will help us monitor and fix things if/when they break and we will be able to tune traffic and lower our costs of operation.
+
+### Goals
+
+A policy and procedure for use by SIG Release to promote container images and release binaries to multiple registries and mirrors.
+
+A solution to allow redirection to appropriate mirrors to lower cost and allow access from any cloud or country globally.
+
+### Non-Goals
+
+Anything related to creation of artifacts, bom, digital signatures, staging buckets.
```

Comment:
Starting 1.24, the releases will be signed. digital signatures are now in scope.

```suggestion
Anything related to creation of artifacts, bom, staging buckets.
```

Reply:
Updated in f960d6f21b89a3d931d430e6d5e5ac18ce041107